paseos/visualization/space_animation.py

```diff
@@ -0,0 +1,269 @@
+from paseos.actors.base_actor import BaseActor
+from paseos.actors.spacecraft_actor import SpacecraftActor
+from paseos.actors.ground_station_actor import GroundstationActor
+from paseos.paseos import PASEOS
+from paseos.visualization.animation import Animation
+
+import pykep as pk
+import numpy as np
+from dotmap import DotMap
+from typing import List
+import matplotlib.pyplot as plt
+from matplotlib.font_manager import FontProperties
+import matplotlib.animation as animation
+from matplotlib.artist import Artist
+
+
+class SpaceAnimation(Animation):
+    """This class visualizes the central body, local actor and known actors over time."""
+
+    fp = FontProperties(fname=r"./FontAwesome.otf")
+    symbols = DotMap(
+        battery_100="\uf240",
+        battery_75="\uf241",
+        battery_50="\uf242",
+        battery_25="\uf243",
+        battery_0="\uf244",
+        battery_charging="\uf376",
+        communication="\uf1eb",
+    )
+
+    def __init__(self, sim: PASEOS) -> None:
```

Comment:
I noticed there is no logging at all in this class. at least some debug output may be nice?